docs/source/tables/CUBLAS_API_supported_by_HIP.md

```diff
@@ -1,6 +1,6 @@
 # CUBLAS API supported by HIP
 
-## **2. CUBLAS Data types**
+## CUBLAS Data types
```

Comment:
The entire CUDA2HIP documentation is autogenerated by hipify-clang. We already agreed on paragraph numbering with Roopa, and this is how these docs are presented on docs.amd.com.